<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingsWidgetMetaData</class>
 <widget class="QWidget" name="DatabaseSettingsWidgetMetaData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="databaseNameLabel">
     <property name="text">
      <string>Database Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="databaseName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="databaseDescriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="databaseDescription"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="databaseDefaultUsernameLabel">
     <property name="text">
      <string>Default Username:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="databaseDefaultUsername"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
